### PR TITLE
Add document for the configuration of the IcingaDB-Modul

### PR DESCRIPTION
**Pull request by RincewindsHat**
This adds a listing of the possible settings for the module in the configuration files.
It is quite similar to the monitoring module, but wasn't documented for this one.

### CODE REVIEW

**Comment by nilmerg** [comment]
Please also add a section to `10-Migration.md` regarding this topic

**Comment by RincewindsHat** [comment]
@nilmerg better?

**Review by nilmerg — CHANGES_REQUESTED** [review]
Please also rebase, to fix the actions

File: doc/10-Migration.md [hunk 1]
```diff
@@ -158,3 +158,16 @@ general access to the monitoring module, this is not automatically migrated. You
 It gives you the chance to review the performed changes, before letting them loose on your users. Please also
 take in mind, that Icinga DB Web handles permissions and restrictions differently. Our blog provides details
 on that: https://icinga.com/blog/2021/04/07/web-access-control-redefined/#icingadb-permission-linkage
+
+## General configuration via config.ini
```
[the remark this answers is not in the document]
Comment (RincewindsHat) [reply doc/10-Migration.md:162]: This is intentionally placed here, it describes how one would migrate that kind of configuration from the monitoring module to the icingadb module

File: doc/10-Migration.md [hunk 2]
```diff
@@ -158,3 +158,16 @@ general access to the monitoring module, this is not automatically migrated. You
...
+
+If some of the default options of the monitoring module (for example setting
+acknowledgements to expire by default) were adjusted via the `config.ini` file,
+migrating those options is done by simple copying the configuration file to the icingadb module.
```
[the remark this answers is not in the document]
Comment (RincewindsHat) [reply doc/10-Migration.md]: fixed

File: doc/10-Migration.md [hunk 3]
```diff
@@ -158,3 +158,16 @@ general access to the monitoring module, this is not automatically migrated. You
...
+In most cases this would be a
+
+```
+cp /etc/icingaweb2/modules/monitoring/config.ini /etc/icingaweb2/modules/icingadb/config.ini
+```
+
+The behaviour of those options remains the same.
```
[the remark this answers is not in the document]
Comment (RincewindsHat) [reply doc/10-Migration.md]: fixed?

File: doc/10-Migration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nilmerg) [reply doc/10-Migration.md:162]: The entire document is about that. The document however has different sections:

* Configuration (General, global stuff)
* Navigation (user specific customization)
* Dashboards (user specific customization)
* Views and Exports (Custom views)
* Access Control

Your addition clearly belongs to the first section. If you'd keep it where it is, the reader would be surprised to find this after reading through the entire document.

File: doc/10-Migration.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nilmerg) [reply doc/10-Migration.md]: Oh come on. This is **not** about command transports :roll_eyes:

[the remark this answers is not in the document]
Comment (RincewindsHat) [reply doc/10-Migration.md]: my oversight. sorry.

File: doc/10-Migration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RincewindsHat) [reply doc/10-Migration.md]: ah yes, my fault. I did misinterpret the structure.

File: doc/03-Configuration.md [hunk 4]
```diff
@@ -61,6 +61,45 @@ For this you need an `ApiUser` object with at least the following permissions on
     also configure the secondary master's API command transport.
     Icinga DB Web then uses this transport if the primary one is not available.
 
+## General Configuration
+
+Some default values for actions with dialogs in the UI can be adjusted (e.g. While acknowledging a problem).
```
Comment (nilmerg) [inline doc/03-Configuration.md:66]: Was something wrong with my suggestion?

You're now talking of *some values* and in the next sentence of *these options*. I'd wonder, which options are meant.

And while there are indeed *actions **without** dialogs*, I don't think this nuance needs to be mentioned here.

Though, I see why you added *some* at the start, as I used *particular* to refer to dialogs myself.

So what about:

```suggestion
You can adjust some default values of options users have while interacting with particular dialogs in the UI. (e.g. While acknowledging a problem)
```

**Review by nilmerg — CHANGES_REQUESTED** [review]

**Review by nilmerg — APPROVED** [review]
Thanks ;)

**Pull request merged**